source/ns_cmdline.c

```diff
@@ -1823,6 +2024,16 @@ int unset_command(int argc, char *argv[])
     cmd_variable_add(argv[1], NULL);
     return 0;
 }
+#if MBED_CONF_CMDLINE_USE_DUMMY_SET_ECHO_COMMANDS == 1
+int echo_command(int argc, char *argv[])
+{
+    return 0;
```

Comment:
Doesn't this give a "unused variable" compiler warning?

Reply:
true. I'll add linux builds compiler flag `-Werror` so these causes CI failure..